### PR TITLE
Code Quality: Refactor InnerNavigationToolbar

### PR DESCRIPTION
**Pull request by 0x5bfa**
### Description

One of the plans to add XAML comments.

### Motivation and Context

- Add XAML comments
- Refactor the code-behind
- Remove ShouldConstrainToRootBounds Property (According to this [MSDN](https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.primitives.popup.shouldconstraintorootbounds?view=windows-app-sdk-1.2))

### PR Checklist
- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
- [x] Did you build the app and test your changes?
- [x] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?

### CODE REVIEW

File: src/Files.App/UserControls/InnerNavigationToolbar.xaml [hunk 1]
```diff
@@ -185,10 +210,12 @@
 					Label="{helpers:ResourceString Name=Properties}"
 					LabelPosition="Collapsed"
 					ToolTipService.ToolTip="{helpers:ResourceString Name=Properties}">
-
-					<local:OpacityIcon Style="{StaticResource ColorIconProperties}" />
```
Comment (yaira2) [inline src/Files.App/UserControls/InnerNavigationToolbar.xaml:189]: ?

File: src/Files.App/UserControls/InnerNavigationToolbar.xaml [hunk 2]
```diff
@@ -330,36 +375,41 @@
...
 				<AppBarButton
 					x:Name="InstallFontButton"
 					x:Load="{x:Bind Commands.InstallFont.IsExecutable, Mode=OneWay, FallbackValue=False}"
 					Command="{x:Bind Commands.InstallFont, Mode=OneWay}"
 					Label="{x:Bind Commands.InstallFont.Label}"
 					LabelPosition="Default"
 					ToolTipService.ToolTip="{helpers:ResourceString Name=Install}">
-
-					<!--  Add icon  -->
+					<!--  TODO: Add Icon  -->
```
Comment (d2dyno1) [inline src/Files.App/UserControls/InnerNavigationToolbar.xaml:409]: TODO?

Comment (0x5bfa) [reply src/Files.App/UserControls/InnerNavigationToolbar.xaml:409]: If it is not todo, what's supposed to be meant 'Add icon'?

File: src/Files.App/UserControls/InnerNavigationToolbar.xaml (hunk 1 above)
Comment (0x5bfa) [reply src/Files.App/UserControls/InnerNavigationToolbar.xaml:189]: It was a mistake, fixed.

File: src/Files.App/UserControls/InnerNavigationToolbar.xaml (hunk 2 above)
Comment (d2dyno1) [reply src/Files.App/UserControls/InnerNavigationToolbar.xaml:410]: @yair100

File: src/Files.App/UserControls/InnerNavigationToolbar.xaml [hunk 3]
```diff
@@ -529,29 +585,35 @@
 				LabelPosition="Collapsed"
 				Style="{StaticResource ToolBarAppBarButtonFlyoutStyle}"
 				ToolTipService.ToolTip="{helpers:ResourceString Name=Layout}">
+
 				<AppBarButton.Icon>
-					<FontIcon FontSize="10" Glyph="&#xE152;" />
```
Comment (Jay-o-Way) [inline src/Files.App/UserControls/InnerNavigationToolbar.xaml:534]: As @yair100 knows, a number of icons are left to update. See #10783 /  #11822

File: src/Files.App/UserControls/InnerNavigationToolbar.xaml (hunk 2 above)
Comment (yaira2) [reply src/Files.App/UserControls/InnerNavigationToolbar.xaml:410]: It's a todo for myself to add an opacity icon.

**Comment by mdtauk** [comment]
Are you not using the custom icons already made, for those layout options?

image

**Comment by yaira2** [comment]
> Are you not using the custom icons already made, for those layout options?

We haven't done any work on that yet. This PR isn't introducing an UI changes.

**Comment by 0x5bfa** [comment]
I did actually. If you want me to update those icons to use opacity icons you newly make, let me know.

**Comment by yaira2** [comment]
@0x5bfa this change hasn't been approved, can you open an issue to discuss it?

**Comment by 0x5bfa** [comment]
@yair100  Ready for review.

> @0x5bfa this change hasn't been approved, can you open an issue to discuss it?

Revarted changes

File: src/Files.App/UserControls/InnerNavigationToolbar.xaml [hunk 4]
```diff
@@ -185,10 +210,13 @@
...
 					LabelPosition="Collapsed"
 					ToolTipService.ToolTip="{helpers:ResourceString Name=Properties}">
-
 					<local:OpacityIcon Style="{StaticResource ColorIconProperties}" />
 				</AppBarButton>
+
+				<!--  (Divider)  -->
 				<AppBarSeparator x:Name="AdditionalActionSeparator" x:Load="{x:Bind ViewModel.HasAdditionalAction, Mode=OneWay}" />
+
+				<!--  Expty Recycle Bin  -->
```
Comment (QuaintMako) [inline src/Files.App/UserControls/InnerNavigationToolbar.xaml:219]:
```suggestion
				<!--  Empty Recycle Bin  -->
```

**Review by QuaintMako — COMMENTED** [review]
I cannot vouch for everything in the `xmal`, but the .`cs` looks in order.
LGTM overall.

**Pull request closed without merging**